.circleci/config.yml

```diff
@@ -40,7 +40,23 @@ aliases:
       mkdir -p ~/.ssh
       echo github.com ssh-ed25519 $GITHUB_SSH_KEY >> ~/.ssh/known_hosts
 
-      git clone --depth 1 "$CIRCLE_REPOSITORY_URL" --branch "$CIRCLE_BRANCH" .
+      # Clone remote and check the commit ID of the checked out code (sh version of bash `elif [[ "$CIRCLE_BRANCH" =~ ^pull\/* ]]`)
```

Comment:
What is this comment trying to say? At first I thought it was talking about the following line, but it seems unrelated.

It's also odd to see something described as the "sh version" of a bash snippet, given that this _is_ a Bash script.

Reply:
That comment was coming from here, but yeah I have revised it now
https://github.com/guitarrapc/git-shallow-clone-orb/blob/main/src/commands/checkout.yml